pkg/authenticator/config/config.go

```diff
@@ -68,9 +99,17 @@ func NewFromEnv() (*Config, error) {
 
 	containerMode := os.Getenv("CONTAINER_MODE")
 
-	conjurVersion := os.Getenv("CONJUR_VERSION")
-	if len(conjurVersion) == 0 {
-		conjurVersion = "5"
+	// Only versions '4' & '5' are allowed, with '5' being used as the default
+	conjurVersion := DefaultConjurVersion
+	switch os.Getenv("CONJUR_VERSION") {
+	case "4":
+		conjurVersion = os.Getenv("CONJUR_VERSION")
```

Comment:
A slight variation, maybe... Instead of calling os.Getenv() twice, you already know what it's going to be:
```
	case "4":
		conjurVersion = "4"
```